src/styles/global.css

```diff
@@ -0,0 +1,3 @@
+#hubspot-messages-iframe-container > iframe {
```

Comment:
this should probably have some sort of comment above it as to why it's been added